.github/workflows/tests.yml

```diff
@@ -211,10 +213,13 @@ jobs:
       matrix:
         platform: [ubuntu-24.04, macos-13, windows-latest]
         python-version: ["3.9", "3.12"]
-        test-suite: [ "chemistry_freezing_isotopes", "condensation_a", "condensation_b", "coagulation", "breakup", "multi-process_a", "multi-process_b"]
+        test-suite: [ "chemistry_freezing_isotopes_a", "chemistry_freezing_isotopes_b", "condensation_a", "condensation_b", "condensation_c", "coagulation", "breakup", "multi-process_a", "multi-process_b", "multi-process_c", "multi-process_d", "multi-process_e"]
+        exclude:
+            # TODO #1469
+            - test-suite: "coagulation"
```

Comment:
this would best be fixed before merging, since coagulation is absolutely central to the whole package, and fixing seems to require some matplotlib trickery only

Reply:
OK, I will look into this and delete exclusion.

Reply:
Thanks!